src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Application/Queries/GetCohortPriorLearningError/GetCohortPriorLearningErrorQueryHandlerTests.cs

```diff
@@ -0,0 +1,298 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.Authorization.Features.Models;
+using SFA.DAS.Authorization.Features.Services;
+using SFA.DAS.CommitmentsV2.Application.Queries.GetDraftApprenticeshipPriorLearningSummary;
+using SFA.DAS.CommitmentsV2.Data;
+using SFA.DAS.CommitmentsV2.Domain.Entities;
+using SFA.DAS.CommitmentsV2.Domain.Interfaces;
+using SFA.DAS.CommitmentsV2.Models;
+using SFA.DAS.CommitmentsV2.Services;
+using SFA.DAS.CommitmentsV2.Types;
+using SFA.DAS.UnitOfWork.Context;
+using TrainingProgramme = SFA.DAS.CommitmentsV2.Domain.Entities.TrainingProgramme;
+
+namespace SFA.DAS.CommitmentsV2.UnitTests.Application.Queries.GetCohortPriorLearningError
+{
+    [TestFixture]
+    [Parallelizable]
+    public class GetCohortPriorLearningErrorQueryHandlerTests
+    {
```

Comment:
this is quite confusing. the class name suggests it's testing GetCohortPriorLearningErrorQueryHandler but it's actually testing GetDraftApprenticeshipPriorLearningSummaryQueryHandler

I'm curious what's happened to the original c# class? I guessed it was called GetDraftApprenticeshipPriorLearningSummaryQueryHandlerTests but can't find it as deleted in the PR. Did I not write any?